packet-exporter/src/main.rs

```diff
@@ -0,0 +1,83 @@
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate prometheus;
+
+use std::error::Error;
+
+use hyper::header::CONTENT_TYPE;
+use hyper::service::{make_service_fn, service_fn};
+use hyper::{Body, Client, Request, Response, Server, Uri};
+use hyper_tls::HttpsConnector;
+use prometheus::{Encoder, GaugeVec, TextEncoder};
+use serde_derive::{Deserialize, Serialize};
```

Comment:
Since we're already using 2018 edition, I'd like to keep the occurrences of `#[macro_use]` as low as possible.

```suggestion
use std::error::Error;

use hyper::header::CONTENT_TYPE;
use hyper::service::{make_service_fn, service_fn};
use hyper::{Body, Client, Request, Response, Server, Uri};
use hyper_tls::HttpsConnector;
use lazy_static::lazy_static;
use prometheus::{register_gauge_vec, Encoder, GaugeVec, TextEncoder};
use serde_derive::{Deserialize, Serialize};
```